template.md

```diff
@@ -20,8 +20,10 @@ Deadline for advancement eligibility: [**CALCULATE TIME HERE**](https://www.time
   - <sub>Note: this time is selected to be precisely 10 days prior to the start of the meeting</sub>
 
 1. Proposals not looking to advance may be added at any time; if after the deadline, please always use a pull request so that members are notified of changes.
-1. Proposals looking to advance to stage 0 or 1 must be added (and noted as such) prior to the deadline, or else delegates may object to advancement solely on the basis of missing the deadline.
+1. Proposals seeking feedback at stage 0 must be added (and noted as such) prior to the deadline, or else delegates may object to advancement solely on the basis of missing the deadline.
```

Comment:
If these are just seeking feedback, does the "object to advancement" statement apply to them?

Reply:
I am not sure what to do with stage 0, we can't object in those cases but im not sure what to do here.

Reply:
Me neither.

Reply:
Something like this, perhaps?
```suggestion
1. Proposals seeking feedback at stage 0 must be added (and noted as such) prior to the deadline, or else delegates may object to the presentation solely on the basis of missing the deadline.
```

Reply:
I don't think anyone can (or should be able to) object to a presentation, generally.

Typically, we've considered something stage 0 when it's either been presented to the committee, or is on the agenda to be, and the only reason something won't qualify for stage 0 (or will be evicted from it) is if there's zero likelihood it will proceed.